src/markdown/setup/tableau-desktop-driver-setup.md

```diff
@@ -17,7 +17,7 @@
 #### Connecting to Amazon DocumentDB Using Tableau - External SSH Tunnel
 1. Launch the Tableau Desktop application.
 
-2. Navigate to **Connect > To A Server > More**. **Amazon DocumentDB by AWS** should be listed
+2. Navigate to **Connect > To A Server > More**. **Amazon DocumentDB by Amazon Web Services** should be listed
```

Comment:
There are 4 other instances of "Amazon DocumentDB by AWS" in the troubleshooting-guide.md that also need to be updated.

Reply:
Good catch!! Totally forgot about these.